### DISCUSSION TITLE
show the first css classname

### DISCUSSION THREAD

**Discussion by chenbimo** [moalamri/vscode-inline-fold#91] [Ideas]
it's not intuitive when show three dot,i hope it have a option,that can set to show the first css classname.

**Comment by moalamri** [comment]
hey @chenbimo 
I can't think of a benefit of showing the first word of the class value since it can't be an identifier to the element, even two or three words 😅 . I would recommend to put commands in use and bind the toggler to keyboard shortcut.
That will put the opened file in two states: styling or markup.

**Reply by chenbimo** [reply]
sometimes,we can set a name for "div",just like a people's name,do not define the css style for it.it is just a symbol or markup,tell us,what is the "div" is.this is my think and idea.very thanks for your reply.happy new year!

**Reply by moalamri** [reply]
Well, if you have a solid idea that I can implement in future updates, that also could be useful in other use-cases (not only tsx className), then I'm open for suggestions. Happy new year 🥳 🎉
